markdown/assignments/pool_concept/tasks/junit_advanced_rsv.md

```diff
@@ -16,7 +16,7 @@ public class MitgliederVerwaltung {
      */
     public boolean testBeitritt(int alter, int motivation) {
         if (alter < 0 || alter > 99 || motivation < 0 || motivation > 10) {
-            throw new IllegalArgumentException("Das Alter und die Motivation müssen innerhalb der Grenzen sein.");
+            throw new IllegalArgumentException("Das Alter und/oder die Motivation ist ungültig.");
```

Comment:
```suggestion
            throw new IllegalArgumentException("Alter oder Motivation ist ungültig");
```

Reply:
@Tobias321 Ich kürze das mal noch weiter ein. Das Problem war der Umbruch im Code-Block in der Webseite, das sah extrem doof aus. Ich will aber auch keine extra Zeile oder so spendieren, und letztlich ist die Message hier nur Nebensache :)